src/js/core/player.js

```diff
@@ -731,7 +731,7 @@ function name() {
     const lnRand = random.uniform(0, playerNames.last[playerNames.last.length - 1][1]);
     const ln = playerNames.last.find(row => row[1] >= lnRand)[0];
 
-    return `${fn} ${ln}`;
+    return { firstName : fn, lastName : ln }
```

Comment:
Are you running the linter? I don't like spaces inside curly brackets here, and it needs a semicolon at the end.

Also... would be slightly cleaner if this line was just:

    return {firstName, lastName};

which is valid ES6 syntax (called object literal shorthand) if you change the `fn` variable to `firstName` and the `ln` variable to `lastName`.